sw/device/silicon_creator/manuf/lib/provisioning.c

```diff
@@ -90,6 +88,95 @@ static_assert(OTP_CTRL_PARAM_CREATOR_ROOT_KEY_SHARE0_SIZE ==
                   OTP_CTRL_PARAM_CREATOR_ROOT_KEY_SHARE1_SIZE,
               "Detected Root key share size mismatch");
 
+// ECC curve to use with ECDH keygen.
+static const ecc_curve_t kCurveP256 = {
+    .curve_type = kEccCurveTypeNistP256,
+    .domain_parameter = NULL,
+};
+
+// ECDH private key configuration.
+static const crypto_key_config_t kEcdhPrivateKeyConfig = {
+    .version = kCryptoLibVersion1,
+    .key_mode = kKeyModeEcdh,
+    .key_length = kP256ScalarBytes,
+    .hw_backed = kHardenedBoolFalse,
+    .diversification_hw_backed = NULL,
+    .security_level = kSecurityLevelLow,
```

Comment:
@moidx @jadephilipoom is this ok for the RMA token export? what is our threat model in this part of the manufacturing flow?

Reply:
Currently this setting actually doesn't do anything. We were considering using it as a switch that allows the caller to turn off e.g. physical-attacker countermeasures if they don't care about physically present attackers. It's less clear now that there's a use case for this, so I might end up just removing it. But in case it stays in and does in the future do something, I'd change it to `kSecurityLevelHigh` in this case since physical attackers do matter here.